src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1550,6 +1550,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("VFR_HUD", 4.0f);
 		configure_stream_local("VIBRATION", 0.1f);
 		configure_stream_local("WIND_COV", 0.5f);
+		configure_stream_local("EFI_STATUS", 2.0f);
```

Comment:
Can you keep the list sorted alphabetically?

Reply:
fixed